src/Foundation/Defaults.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Comparables.swift
+//
+//  Created by Logan Wright on 10/25/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+import Foundation
+
+protocol SettingsKeyAccessible {
+    var key: String { get }
+    var defaults: NSUserDefaults { get }
+}
+
+extension SettingsKeyAccessible {
+    var defaults: NSUserDefaults {
```

Comment:
I would make this a convenience initializer with nsuserdefaults to be passed in as the default so we pave the way for testing